### DISCUSSION TITLE
Control for Embed Spaces

### DISCUSSION THREAD

**Discussion by xBelabz** [GitbookIO/community#197] [Feature Requests]
### What's your feature/idea?

Today we are sharing GitBook with Embed links into our LMS.
However the ellipses icon is still showing, it exposes the copy link button that has no sense in this EMbed context.

It would be great to control which options we would like to show in the spaces.

Keep Rocking.

### What's your desired outcome?

Protect my Emned content link as much as possible we can.

### What's the impact of this for you?

- [ ] Critical
- [X] High
- [ ] Medium
- [ ] Low

### Anything else?

A control for the Dark / Light mode could be a great feature to be added for the readers.
Today only the org or the creator decides in which mode he would show his content.

**Comment by addisonschultz** [comment]
Hi @xBelabz! 

Addison here from GitBook - we’re working on a few things related to this feature request :)

Can you reach out to me at addison (@) gitbook.com with the Subject on “Embeddable GitBook”?

Would love to chat more!
